docs/requirements.txt

```diff
@@ -1,14 +1,5 @@
-# don't pin crate version numbers so the latest will always be pulled when you
-# set up your environment from scratch
+setuptools>=41.1.0
+sphinx>=1.8.5,<4
+sphinx-autobuild
```

Comment:
this should just be `crate-docs-theme`. the rest is handled by the Makefile or RTD, respectively

see https://github.com/crate/crate-docs/blob/main/docs/requirements.txt

unless there is a good reason to deviate, https://github.com/crate/crate-docs/blob/main/docs should be considered the template project for the common build :)

Reply:
Thanks. Addressed it.